pageserver/src/tenant.rs

```diff
@@ -4996,4 +4996,23 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn test_read_at_max_lsn() -> anyhow::Result<()> {
+        let harness = TenantHarness::create("test_get_vectored")?;
```

Comment:
This uses the same test directory as `test_get_vectored` and results in some strange CI failures.
Sorry for not spotting it the first time around.

```suggestion
        let harness = TenantHarness::create("test_read_at_max_lsn")?;
```

Reply:
Aha, I assumed the CI was flaky since the same test passed locally, thanks